.github/workflows/coveralls.yml

```
@@ -0,0 +1,39 @@
+name: Tests and coveralls
+
+on: ["push", "pull_request"]
+
+jobs:
+  build:
+
+    name: Build
+    runs-on: ubuntu-latest
+
+    steps:
+
+    # Check out Housekeeper code
+    - uses: actions/checkout@v1
+
+    # Set up python
+    - name: Set up Python ${{ matrix.python-version}}
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+
+    - name: Production Dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -e .
```

```suggestion
        pip install .
```
remove -e to make it more production like

ditto in the following installs